client/ayon_core/pipeline/workfile/utils.py

```diff
@@ -2,6 +2,10 @@
 from ayon_core.settings import get_project_settings
 
 
+class MissingWorkdirError(Exception):
+    pass
```

Comment:
I want to avoid the confusion that this may be something else than the directory not existing on disk, e.g. someone mistakenly assuming this may mean something of 'workdir' is missing in the database or whatsoever.

```suggestion
    """Raised when accessing a work directory not found on disk."""
    pass
```